### ISSUE TITLE
PERFORMANCE: Use rowAnyNAs(x) instead of (rowSums(is.na(x)) > 0)

### ISSUE THREAD

**Issue by HenrikBengtsson** [pneuvial/c3co#16]
The following `rowSums(is.na(x)) > 0` statements can be replaced by `rowAnyNAs(x)` of matrixStats (>= 0.52.0):
```r
$ grep -F "rowSums(" R/segmentData.R 
            idxNAC1 <- which(rowSums(is.na(binDatTCN)) > 0)
            idxNAC2 <- which(rowSums(is.na(binDatDH)) > 0)
            idxNA <- which(rowSums(is.na(binDatTCN)) > 0)
```

**Comment by pneuvial** [comment]
Fixed [develop e63f2e0]

**Comment by pneuvial** [comment]
matrixStats (>= 0.52.0) not avaliable on some Travis CI instances and I don't know how to force install:


https://travis-ci.org/pneuvial/c3co/jobs/218813565

**Comment by HenrikBengtsson** [comment]
It can take up to a week before macOS binaries are available on CRAN. The only trick I know is to install from GitHub; fixed in commit ce66c37.

**Comment by HenrikBengtsson** [comment]
Reverted last commit; matrixStats 0.52.1 macOS binaries are now on CRAN.
